js/bitget.js

```diff
@@ -176,6 +177,11 @@ module.exports = class bitget extends Exchange {
                             'wallet/transfer': 4,
                             'wallet/withdrawal': 4,
                             'wallet/subTransfer': 10,
+                            'plan/placePlan': 1,
+                            'plan/modifyPlan': 1,
+                            'plan/cancelPlan': 1,
+                            'plan/currentPlan': 1,
+                            'plan/historyPlan': 1,
```

Comment:
aren't we missing `fetchClosedOrders` ? 
https://bitgetlimited.github.io/apidoc/en/spot/#get-history-plan-orders

Reply:
@carlosmiei I updated fetchClosedOrders, please take a look when you have time.